#Base
logger/Event base
#battery status
logger/BatteryStatus battery
